pkg/storage/snapshot/snapshot.go

```diff
@@ -266,6 +268,13 @@ func (ctrl *VMSnapshotController) unfreezeSource(vmSnapshot *snapshotv1.VirtualM
 	return nil
 }
 
+func generateFinalizerPatch(test, replace []string) ([]byte, error) {
```

Comment:
nit: do we really need this function? You can simply replace it with the actual functions

Reply:
I think it's kind of nice to not have a bunch of hardcoded references to "/metadata/finalizers". Of course could make that a const. Or why not just make a function? VM controller does the same thing: https://github.com/kubevirt/kubevirt/blob/ecc7c8f091bad3c685d5580e6fae18858012c459/pkg/virt-controller/watch/vm/vm.go#L2318-L2323